.travis.yml

```diff
@@ -1,29 +1,15 @@
-language: node_js
-node_js:
-  - "6"
+sudo: required
 
-# This (sudo: false) is needed to "run on container-based infrastructure" on
-# which cache: is available
-# http://docs.travis-ci.com/user/workers/container-based-infrastructure/
-sudo: false
+language: python
 
-before_install:
-  - pip install --user restructuredtext_lint
-  - pip install --user flake8 pycodestyle pep8-naming
-  - flake8 -v .
-  - rst-lint README.rst
+python:
+  - "2.7"
+
+services:
+  - docker
 
-before_script:
-  - npm install -g grunt-cli
+before_install:
+  - git clone --depth=50 --branch=web-apps git://github.com/jburel/omero-test-infra .omero
```

Comment:
Is this likely to get merged back to ```openmicroscopy/omero-test-infa``` before this PR is merged, or will we update this in a future PR?

Reply:
test-infra PR must go in first then I will fix the link